.ci/install_runtime.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cidir=$(dirname "$0")
+
+source "${cidir}/lib.sh"
+
+# Modify the runtimes build-time defaults
+
+# enable verbose build
+export V=1
+
+# tell the runtime build to use sane defaults
+export SYSTEM_BUILD_TYPE=kata
+
+# The runtimes config file should live here
+export SYSCONFDIR=/etc
+
+# Artifacts (kernel + image) live below here
+export SHAREDIR=/usr/share
+
+runtime_config_path="${SYSCONFDIR}/kata-containers/configuration.toml"
+
+PKGDEFAULTSDIR="${SHAREDIR}/defaults/kata-containers"
+NEW_RUNTIME_CONFIG="${PKGDEFAULTSDIR}/configuration.toml"
+# Note: This will also install the config file.
+clone_build_and_install "github.com/kata-containers/runtime"
+
+# Check system supports running Kata Containers
+kata-runtime kata-check
+
+if [ -e "${NEW_RUNTIME_CONFIG}" ]; then
+	# Remove the legacy config file
+	sudo rm -f "${runtime_config_path}"
+
+	# Use the new path
+	runtime_config_path="${NEW_RUNTIME_CONFIG}"
+fi
+
+echo "Enabling all debug options in file ${runtime_config_path}"
+sudo sed -i -e 's/^#\(enable_debug\).*=.*$/\1 = true/g' "${runtime_config_path}"
+
+echo "Add runtime as a new/default Docker runtime. Docker version \"$(docker --version)\" could change according to updates."
+docker_options="-D --add-runtime kata-runtime=/usr/local/bin/kata-runtime"
+
+config_path="/etc/systemd/system/docker.service.d/"
+sudo mkdir -p ${config_path}
+
+# Check if the system has set http[s] proxy
+if [ ! -z "$http_proxy" ] && [ ! -z "$https_proxy" ] ;then
+	docker_http_proxy="HTTP_PROXY=$http_proxy"
+	docker_https_proxy="HTTPS_PROXY=$https_proxy"
+fi
+
+cat << EOF | sudo tee ${config_path}/clear-containers.conf
```

Comment:
s/clear-containers/kata-containers/.

But shouldn't we be using https://github.com/clearcontainers/tests/blob/master/cmd/container-manager/manage_ctr_mgr.sh here instead (in other words import that cmd on this PR too)?